src/main/java/com/targomo/client/api/geo/AbstractGeometry.java

```diff
@@ -51,6 +60,13 @@ public void setCrs(Integer crs) {
         this.crs = crs;
     }
 
+    /**
+     * @param routeFromCentroid if true route from/to the centroid of this geometry when no intersections found
+     */
+    public void setRouteFromCentroid(Boolean routeFromCentroid) {
```

Comment:
I think we should use the primitive `boolean` here

Reply:
switched